### PR TITLE
added my details on linkFree

### PR DESCRIPTION
**Pull request by yrncollo**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5037"></a>

### CODE REVIEW

File: data/yrncollo.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  "name": "YrnCollo",
+  "type": "personal",
+  "bio": "Open Source Contributor | Cyber Security researcher| Reverse engineer | Red teamer",
+
+  "tags": [
+    "Open Source",
+    "C++",
+    "Python",
+    "C",
+    "VimScript",
```
Comment (SurajPratap10) [inline data/yrncollo.json]: `"VimScript",`  remove the comma

**Review by SurajPratap10 — APPROVED** [review]
cool

File: data/yrncollo.json (hunk 1 above)
Comment (yrncollo) [reply data/yrncollo.json]: Thanks I have already done that

File: data/yrncollo.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/Yrn_Collo",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Blog: Let's share ideas",
+      "url": "https://yrncollo.github.io/",
+      "icon": "FaMicroBlog",
```
Comment (amandamartin-dev) [inline data/yrncollo.json:30]: this should be FaMicroblog
https://react-icons.github.io/react-icons/search?q=microblog

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/yrncollo.json (hunk 2 above)
Comment (yrncollo) [reply data/yrncollo.json]: I have just updated it

**Review by amandamartin-dev — APPROVED** [review]

**Review by iamthecloverly — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**